src/app/games/abalone/abalone.component.ts

```diff
@@ -80,16 +79,15 @@ export class AbaloneComponent extends HexagonalGameComponent<AbaloneRules,
         this.directions = [];
         this.selecteds = [];
     }
-    private showPreviousMove(): void {
-        const move: AbaloneMove = this.rules.node.move;
+    private showPreviousMove(move: AbaloneMove): void {
```

Comment:
uniformity in showPreviousMove (like mentionned somewhere else)

Reply:
done